en/components/grid-summaries.md

```diff
@@ -40,6 +40,9 @@ When one or more Summary objects is added to the Sketch diagram, those summaries
 
 <img src="../images/grid_summaries_codegen.png" />
 
+> [!Note]
+> There is an issue with the Ignite UI Summary Theme rendering, the background color function does not render the color, instead the Header color gets rendered. When this is corrected the correct color from the Sketch drawing will be rendered properly.
```

Comment:
This is kind of a runon sentence.  It makes sense, but it's hard to parse.  I'd recommend splitting it into 3 sentences.
There is an issue with the Ignite UI Summary Theme rendering.
The background color function does not render the color, but instead renders the Header color.
When this is corrected, the correct color from the Sketch drawing will be rendered properly.
Also, in that last sentence, it seems repetitive.  "corrected", "correct", "properly" kind of all mean the same thing.  Maybe "When this is fixed, the color from the Sketch drawing will be rendered."